### PR TITLE
Cgi proposal2

### PR DESCRIPTION
**Pull request by chris-giblin**

### CODE REVIEW

File: 015-kafka-topic-encryption.md [hunk 1]
```diff
@@ -97,38 +93,37 @@ Envoy however is but one viable approach, but certainly not the only means, to e
 in a proxy.
```
Comment (zrlvep) [inline 015-kafka-topic-encryption.md:93]: "Envoy however is but one viable approach, but certainly not the only means, to embed topic encryption in a proxy." Is that a correct English sentence? Sounds strange in my non-native ears.

File: 015-kafka-topic-encryption.md [hunk 2]
```diff
@@ -97,38 +93,37 @@ Envoy however is but one viable approach, but certainly not the only means, to e
...
-Call out the projects in the Strimzi organization that are/are not affected by this proposal. 
-
+This proposal does not conflict or overlap with specific Strimzi projects but rather
+provides a complementary, new functionality which ideally will become part of the
+Strimzi Kafka distribution.
 
 ## Compatibility
 
-Call out any future or backwards compatibility considerations this proposal has accounted for.
+As this is a new capability, there are no backwards compatibility issues relating to this proposal.
```
Comment (zrlvep) [inline 015-kafka-topic-encryption.md]: I'd add something like "The system will intercept ProduceRequests and FetchResponses and therefore must make sure that any future changes to the Kafka API in these specific message types are accounted for."

**Review by zrlvep — APPROVED** [review]
Just some minor remarks.

File: 015-kafka-topic-encryption.md (hunk 1 above)
Comment (chris-giblin) [reply 015-kafka-topic-encryption.md:93]: the English is correct. Stylistically there should be one "but". I will update.

File: 015-kafka-topic-encryption.md (hunk 2 above)
Comment (chris-giblin) [reply 015-kafka-topic-encryption.md]: Great point. I just pushed a new version and check that I captured your suggestion correctly.

File: 015-kafka-topic-encryption.md (hunk 1 above)
Comment (chris-giblin) [reply 015-kafka-topic-encryption.md:93]: Just pushed a revised wording

Comment (zrlvep) [reply 015-kafka-topic-encryption.md:93]: Hm, to me it still sounds strange. I would have removed the second "but". But I'm not the one who can judge this :)

**Review by zrlvep — APPROVED** [review]
lgtm

**Review by SeanRooooney — APPROVED** [review]
I think we are good to go.

**Pull request merged**